### PR TITLE
LPS-163706 Add all workflow fields to AccountEntry

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-163706](https://issues.liferay.com/browse/LPS-163706).

TODO:

- [ ] Write integration tests

### CODE REVIEW

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountEntryDisplay.java [hunk 1]
```diff
@@ -14,130 +14,36 @@
...
  * @author Pei-Jung Lan
  */
-public class AccountEntryDisplay {
-
-	public static AccountEntryDisplay of(AccountEntry accountEntry) {
-		if (accountEntry != null) {
-			return new AccountEntryDisplay(accountEntry);
-		}
-
-		return new AccountEntryDisplay();
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountEntryDisplay.java:64]: Sorry I might have missed it, but what happens to this default AccountEntryDisplay object after the refactor? I believe this is used when creating a new accountEntry.

**Comment by melodywu** [comment]
ci:test:account

**Comment by melodywu** [comment]
ci:test:bundle

**Comment by patriciajeanperez** [comment]
Hey @drewbrokke,

Can you please take a look at this issue? The domains do not display under the Valid Domains section in the Edit Account page after saving. They are applied but the table with the domains do not show up.

`java.lang.ClassCastException: com.liferay.account.admin.web.internal.display.AccountEntryDisplay cannot be cast to java.lang.String`

Thank you!

**Comment by drewbrokke** [comment]
Superceded by https://github.com/liferay-usm/liferay-portal/pull/791

**Pull request closed without merging**